### DISCUSSION TITLE
q-select label not shown when selected option is too short

### DISCUSSION THREAD

**Discussion by AoooR42** [quasarframework/quasar#13355] [General - Components / Directives / etc] [answered]
Hello, I have a problem with q-select. It has a label, but the size of the q-select block only depends on the option selected, so if there is no option selected or if the option selected is too short, then the label will not be shown (or partially). How can I set the minimal size of the q-select block depending on the label length (the label length is not fixed) ?
Thank you ! :)

**Comment by metalsadman** [comment, marked as answer]
you can set a `style="min-width:100px"` in your QSelect tag.
